src/sentry/static/sentry/app/components/idBadge/projectBadge.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {AvatarProject} from 'app/types';
+import BaseBadge from 'app/components/idBadge/baseBadge';
+import BadgeDisplayName from 'app/components/idBadge/badgeDisplayName';
+
+type Props = {
+  project: AvatarProject;
+  avatarSize?: number;
+  className?: string;
+  hideOverflow?: boolean | string;
+  // Inherited from BaseBadge
```

Comment:
Why can't you pull these props types in from BaseBadge?

Reply:
It hasn't been converted to typescript yet. It is next on my list though.